### PR TITLE
Introducing INTN_T types into tcc parser

### PR DESCRIPTION
**Pull request by XVilka**
Because int/short/long types have different length for different platforms, it makes more sense to use int8_t, int16_t, int32_t, int64_t everywhere. I'll improve other parts of tcc as well.

### CODE REVIEW

**Comment by XVilka** [comment]
Ignore the indentation - TCC originally had broken one.

File: shlr/tcc/libtcc.c [hunk 1]
```diff
@@ -404,20 +340,24 @@ static int tcc_compile(TCCState *s1)
     anon_sym = SYM_FIRST_ANOM;
 
 	/* define some often used types */
-    int_type.t = VT_INT;
-	llong_type.t = VT_LLONG;
+	int8_type.t = VT_INT8;
+	int16_type.t = VT_INT16;
+    int32_type.t = VT_INT32;
```
Comment (radare) [inline shlr/tcc/libtcc.c]: use signal, use tabs

**Comment by XVilka** [comment]
As I told - tcc indentation is broken, check the code not the indent, I'll reindent whole TCC after this PR.

**Comment by XVilka** [comment]
Added fixed tests PR in radare2-regressions https://github.com/radare/radare2-regressions/pull/1067

**Pull request merged**